fontir/src/orchestration.rs

```diff
@@ -30,6 +30,43 @@ bitflags! {
     }
 }
 
+pub type ContextItem<T> = Arc<RwLock<Option<Arc<T>>>>;
```

Comment:
to clarify, I think it would be nice if this was an actual struct, and then you could just have `get` and `set` methods (or whatever else you need) and could hide the details of acquiring the lock etc?